string name
string col
---
bool success